.github/workflows/unittest.yml

```diff
@@ -34,6 +34,7 @@ jobs:
         mkdir -p $HOME/.gradle
         echo "org.gradle.jvmargs=-Xmx4096M" >> $HOME/.gradle/gradle.properties
```

Comment:
maybe it worth to add `Xms` parameter to some higher value (that's minimum memory, or in other words starting memory). Otherwise if it starts with 256M, it will resize/reallocate memory more times. Say 2048, I think we can afford it